src/__tests__/Layout-test.js

```diff
@@ -905,8 +905,8 @@ describe('Layout', function() {
           {style: {margin: 20, measure: text(texts.big)}}
         ]}
       ]},
-      {width: 200, height: 76, top: 0, left: 0, children: [
-        {width: 200, height: 76, top: 0, left: 0, children: [
+      {width: 200, height: 72, top: 0, left: 0, children: [
+        {width: 200, height: 72, top: 0, left: 0, children: [
```

Comment:
I replaced this 72 constant with textSizes.bigHeight + 40, this way it'll won't fail with the next update